Cargo.toml

```diff
@@ -20,6 +20,7 @@ specialization = []
 may_dangle = []
 drain_filter = []
 drain_keep_rest = ["drain_filter"]
+impl_bincode = ["bincode", "unty"]
```

Comment:
Please change the feature name to `bincode` and use `dep:` syntax:

```toml
bincode = ["dep:bincode", "dep:unty"]
```

Reply:
Happy to do this, but I went with this approach as `Cargo.toml` says the MSRV is 1.57 and "dep:" syntax was introduced in [1.60](https://blog.rust-lang.org/2022/04/07/Rust-1.60.0.html#new-syntax-for-cargo-features).

Reply:
Got it.  No change needed, then.